src/components/AddItemForm/AddItemForm.jsx

```diff
@@ -0,0 +1,267 @@
+import { useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import * as yup from 'yup';
+import { useState } from 'react';
+import { Input, TextArea, SubmitButton, ListBox, ComboBox } from '../Forms';
+
+const schema = yup.object().shape({
+  title: yup.string().required('Please insert your Item Name.'),
+  price: yup.number().positive('Please insert a positive number.'),
+  description: yup.string().required('Please add a description.'),
+  email: yup
+    .string()
+    .email('Should be a valid email.')
+    .required('Please insert your Email.'),
+  country: yup.string().required('Please insert your Country name.'),
+  city: yup.string().required('Please insert your City name.'),
+  file: yup
+    .mixed()
+    .test('required', 'Please upload a photo.', (photo) => {
+      if (!photo.length) return false;
+      return true;
+    })
```

Comment:
Let's not make a picture required, but instead have a placeholder picture in case the user does not add one.

Reply:
It doesnt yeah, but it also doesnt make sense to make them required to add one either.